src/components/common/TemperatureBar.tsx

```diff
@@ -0,0 +1,31 @@
+import { Flex, Progress, Text, ProgressProps } from '@chakra-ui/react';
+
+type TemperatureBarProps = ProgressProps & {
+  value: number;
+  size?: string;
+};
```

Comment:
```suggestion
type TemperatureBarProps = ProgressProps & {
  size?: string;
};
```

`ProgressProps`안에 `value` 속성이 있어서 type에 넣지 않아도 될 것 같습니다!!